### ISSUE TITLE
Some annotations missing

### ISSUE THREAD

**Issue by mhaghighat** [cesarandreslopez/open-images-dataset-pascal-voc#2]
Thank you for providing the annotations in Pascal VOC format.
I noticed that the number of annotations in the provided files are less than the number of images in each set. For example, the validation set has 41,620 images, but only 35,925 XML files are in the https://openimagesvoc.datoscientific.com/open_imagesv4_val_annotations.tar.gz. Some XML annotations are also missing the train and test sets.

**Comment by AtriSaxena** [comment]
Thank for providing the annotations in Pascal VOC format. 
I am facing the problem as many train images don't have their XML file. Please release the code so that we can create XML for missing files. 
Thanks in advance.

**Comment by AtriSaxena** [comment]
Created this simple script to make XML into Pascal VOC format. @mhaghighat 
https://github.com/AtriSaxena/OIDv4_to_VOC
